docs/sample_config.yaml

```diff
@@ -852,6 +855,15 @@ log_config: "CONFDIR/SERVERNAME.log.config"
 #  remote:
 #    per_second: 0.01
 #    burst_count: 3
+#
+#rc_send_to_device:
+#  per_second: 0.1
+#  burst_count: 3
+#
+#rc_federation:
+#  room_key_request:
+#    per_second: 0.1
+#    burst_count: 3
```

Comment:
We only need one of these? Part of me feels like we shouldn't even bother to document it, its very very niche.

Reply:
It seemed that the client API and federation API would want different rates, but combining them would be a bit easier.

Reply:
Oh, the sample config hasn't been updated, which is why I got confused. I think if its ratelimited by sender device it shouldn't really matter tbh